src/menus/image.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  TrashIcon,
+  AlignLeftIcon,
+  AlignRightIcon,
+  AlignCenterIcon,
```

Comment:
I've now created some new icons to use for this – https://github.com/outline/outline-icons/commit/6e7bf9498a8e3271237ef7538a09750ebec02a16

You'll just need to bump the `outline-icons` dep to `v1.22.0` to get them – otherwise perfect implementation here 👌

Reply:
I added the icons - looks great, though I now get these warnings:

<img width="705" alt="Screenshot 2021-01-02 at 06 50 17" src="https://user-images.githubusercontent.com/239305/103451638-cd81bf00-4cc6-11eb-93fa-9f7dfb3b2e63.png">

Reply:
Ah damn, need a better linter setup in the icons repo – I'll fix that up in a patch, they should display okay in the meantime

Reply:
Actually I can see the issue, will send a PR to outline-icons.